packages/host/tests/integration/components/ai-assistant-panel/codeblocks-test.gts

```diff
@@ -98,7 +98,7 @@ module('Integration | ai-assistant-panel | codeblocks', function (hooks) {
       @field firstName = contains(StringField);
       @field firstLetterOfTheName = contains(StringField, {
         computeVia: function (this: Person) {
-          return this.firstName[0];
+          return this.firstName?.[0];
```

Comment:
I'm not sure why we are starting to see this now--but i got an error trying to render this card due to this error. computed's should be written such that we support any referenced properties potentially being undefined as that is a totally valid scenario for any field. It would be nice if TS could enforce this, and we spend lots of time trying to get the types to work out for this a long while back but it was a crazy difficult slog and we never got there.

Reply:
ah i see why this happened. I had a bug where my `Store.isLoaded` property was not being set at the right time and we started rendering the card before the data had completed being loaded into the instance. but still--this is actually a more correct way to author the card.